ci/test-checks.sh

```diff
@@ -84,12 +84,18 @@ cargo_audit_ignores=(
   # Blocked on predicates v1.0.6 removing its dependency on `difference`
   --ignore RUSTSEC-2020-0095
 
+  # generic-array: arr! macro erases lifetimes
+  #
+  # Blocked on libsecp256k1 releasing with upgraded dependencies
+  # https://github.com/paritytech/libsecp256k1/issues/66
+  --ignore RUSTSEC-2020-0146
+
 )
 _ scripts/cargo-for-all-lock-files.sh +"$rust_stable" audit "${cargo_audit_ignores[@]}"
 
 {
   cd programs/bpf
-  _ "$cargo" stable audit
+  _ "$cargo" stable audit "${cargo_audit_ignores[@]}"
```

Comment:
@jackcmay no issue with ignore here, right? In fact, is this even necessary with audit called via ./scripts/cargo-for-all-lock-files.sh above?

Reply:
@t-nelson I don't think we need to audit programs/bpf

Reply:
> @t-nelson I don't think we need to audit programs/bpf

Thanks!  I'll pull it out in #15597